internal/core/metadata/rest_deviceprofile.go

```diff
@@ -20,10 +20,14 @@ import (
 	"net/http"
 	"net/url"
 
-	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 	"github.com/gorilla/mux"
 	"gopkg.in/yaml.v2"
+
+	errors2 "github.com/edgexfoundry/edgex-go/internal/core/metadata/errors"
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/device"
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/device/profile"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
```

Comment:
Sorting's a little weird here. See other comments as well. All github.com imports should be alphabetized. Ignore aliases.